src/pages/question/questionBuilder/useNode.js

```diff
@@ -82,9 +75,12 @@ export default function useNodePanels() {
   }
 
   function updateFilteredConcepts(value) {
-    const newFilteredConcepts = conceptsWithSets.filter(
-      (concept) => concept.name.includes(value.toLowerCase()),
-    );
+    // Convert name to lowercase before searching
+    const newFilteredConcepts = concepts
+      .map((c) => ({ ...c, name: c.name.toLowerCase() }))
+      .filter(
+        (concept) => concept.name.includes(value.toLowerCase()),
```

Comment:
I would change this line to (concept) => concept.name.toLowerCase().includes(value.toLowerCase()), because right now you're looping over the list twice but you can do it just once and get rid of the map on line 80.